src/openms/source/KERNEL/MSExperiment.cpp

```diff
@@ -313,14 +313,6 @@ namespace OpenMS
     // update intensity, m/z and RT according to chromatograms as well:
     for (ChromatogramType& cp : chromatograms_)
     {
-
-      // ignore TICs and ECs (as these are usually positioned at 0 and therefor lead to a large white margin in plots if included)
```

Comment:
Why get rid of this?

Reply:
Inconsistent handling of TICs. Kind of a hack that should not be done at this level

Reply:
agreed. lets get rid of it and deal with TOPPView later